Extension/package.json

```diff
@@ -2361,6 +2361,42 @@
           "description": "%c_cpp.configuration.doxygen.generatedStyle.description%",
           "scope": "resource"
         },
+        "C_Cpp.doxygen.selectDoxygenTags": {
```

Comment:
Sorry I missed this earlier.  I think we need a better name for the setting.  Maybe `C_Cpp.doxygen.sectionTags`